docs/source/conf.py

```diff
@@ -341,6 +341,11 @@
 # texinfo_no_detailmenu = False
 
 # enable __init__ method
-autoclass_content = 'both'
-
+autoclass_content = 'class'
+autodoc_default_options = {
+    'special-members': '__init__',
+    'members': True,
+    'undoc-members': True,
+    'show-inheritance': True
+}
```

Comment:
By declaring this variable,
- refactor .rst files
- show `__init__` function in the docs